src/framework/shortcuts/internal/shortcutsregister.cpp

```diff
@@ -192,6 +200,25 @@ void ShortcutsRegister::expandStandardKeys(ShortcutList& shortcuts) const
     }
 }
 
+ShortcutList ShortcutsRegister::updateAdditionalShortcuts(const ShortcutList& shortcuts)
```

Comment:
Probably its name should be removeAdditionalShortcutsFromList

Reply:
No, because this method not only remove from list, it also update additional shortcuts